pyfr/integrators/base.py

```diff
@@ -130,6 +134,38 @@ def cfgmeta(self):
         else:
             return {'config': cfg, 'config-0': cfg}
 
+    def grad_pvars(self, pnames):
```

Comment:
I'd also remove the pnames bit, too.  Let each plugin handle messing with the data as it sees fit.

Reply:
Done